assets/javascripts/discourse/connectors/topic-footer-main-buttons-before-create/assign-button.js.es6

```diff
@@ -3,7 +3,7 @@ import { ajax } from 'discourse/lib/ajax';
 
 export default {
   shouldRender(args, component) {
-    const needsButton = component.currentUser && component.currentUser.get('staff');
+    const needsButton = component.currentUser && component.currentUser.get('staff') && component.siteSettings.assign_enabled;
```

Comment:
Would be great to get to those too eventually. I just thought you could remove the connector specific stuff for this PR :)